src/api/app/views/webui2/webui/users/tasks/index.html.haml

```diff
@@ -0,0 +1,87 @@
+:ruby
+  @pagetitle = "Tasks for #{User.current}"
+  @layouttype = 'custom'
+  @crumb_list = [@pagetitle]
+
+.container
+  .card#reviews
+    .bg-light
+      %ul.nav.nav-tabs.pt-2.px-3.flex-nowrap.collapsible{ 'role': 'tablist' }
+        %li.nav-item
+          - cache "#{User.current.cache_key}-reviews-in" do
+            %a{ href: '#reviews-in', title: "Requests that #{User.current.login} has to review", class: 'nav-link text-nowrap active' }
```

Comment:
drop `.login` it is in the `to_s` method! :see_no_evil: 

[nitpick] or even better, you could use `User#name` here :wink:

Reply:
I would keep using the `login` name for now. That's what we use elsewhere, and in the ui, and that's how users can identify each other, e.g. via the user show page.

Reply:
We have decided to keep all of them as `"#{User.current}"` since `User#to_s` is calling `User#login` and not use `User#name` for the moment.